app/forms/sufia/forms/work_form.rb

```diff
@@ -4,6 +4,7 @@ class WorkForm < CurationConcerns::Forms::WorkForm
     include HydraEditor::Form::Permissions
 
     attr_reader :agreement_accepted
+    attr_accessor :file_sets
```

Comment:
The attribute is already in the object by some magic. I don't know what writes it. I just needed to change it from private to public so that I could access it in the view.

I've got a demo to get ready for tomorrow. I'll get back to this Wednesday. I haven't been able to get my environment set up to run the tests yet.